apps/model/lib/model/route_pattern.ex

```diff
@@ -22,8 +23,9 @@ defmodule Model.RoutePattern do
           direction_id: Model.Direction.id(),
           name: String.t(),
           time_desc: String.t() | nil,
-          typicality: 0..4,
+          typicality: 0..5,
```

Comment:
suggestion: it looks like the documentation in `route_pattern_controller.ex` also needs to be updated with this new value.